plaso/filters/parser_filter.py

```diff
@@ -8,14 +8,16 @@ class ParserFilterExpressionHelper(object):
   """Helper for parser and plugin filter expressions.
 
   A parser filter expression is a comma separated value string that denotes
-  a list of parser names to include and/or exclude. Each entry can have
-  the value of:
-
-  * An exact match of a preset, which is a predefined list of parsers
-    and/or plugins (see data/presets.yaml for the list of predefined
-    presets).
-  * A name of a single parser (case insensitive), such as msiecf.
-  * A glob name for a single parser, such as '*msie*' (case insensitive).
+  which parsers should be used. Each value can contain either:
```

Comment:
parsers => parsers and plugins

since you were quite amendment in https://github.com/log2timeline/plaso/pull/2666 about this